### PR TITLE
deployment: Add Mozilla CA certificate store 2019-08-28

### PR DESCRIPTION
**Pull request by theopolis**
This is the cacerts file that was once provided by the OpenSSL Homebrew
formula. The osquery packages in 3.x and previous provided this store as
the default / fallback.

The hash can be corroborated via:
https://curl.haxx.se/ca/cacert.pem.sha256

38b6230aa4bee062cd34ee0ff6da173250899642b1937fc130896290b6bd91e3

### CODE REVIEW

File: cmake/packaging.cmake [hunk 1]
```diff
@@ -149,6 +149,9 @@ function(generateInstallTargets)
     file(COPY "${CMAKE_SOURCE_DIR}/packs" DESTINATION "${CMAKE_BINARY_DIR}/package/linux")
     install(DIRECTORY "${CMAKE_BINARY_DIR}/package/linux/packs" DESTINATION share/osquery)
 
+    file(COPY "${CMAKE_SOURCE_DIR}/tools/deployment/certs.pem" DESTINATION "${CMAKE_BINARY_DIR}/package/linux")
```
Comment (Smjert) [inline cmake/packaging.cmake]: There's no need to copy the file in the binary folder here. That folder exists only for files that has to be processed in  some way before being consumed by the install step.
Just install the file directly from source.

File: cmake/packaging.cmake [hunk 2]
```diff
@@ -193,6 +196,9 @@ function(generateInstallTargets)
     file(COPY "${CMAKE_SOURCE_DIR}/packs" DESTINATION "${CMAKE_BINARY_DIR}/package/wix")
     install(DIRECTORY "${CMAKE_BINARY_DIR}/package/wix/packs" DESTINATION .)
 
+    # certs
+    file(COPY "${CMAKE_SOURCE_DIR}/tools/deployment/certs.pem" DESTINATION "${CMAKE_BINARY_DIR}/package/wix")
```
Comment (Smjert) [inline cmake/packaging.cmake]: Same

File: cmake/packaging.cmake [hunk 3]
```diff
@@ -213,6 +219,9 @@ function(generateInstallTargets)
     file(COPY "${CMAKE_SOURCE_DIR}/packs" DESTINATION "${CMAKE_BINARY_DIR}/package/pkg")
     install(DIRECTORY "${CMAKE_BINARY_DIR}/package/pkg/packs" COMPONENT osquery DESTINATION /private/var/osquery)
 
+    file(COPY "${CMAKE_SOURCE_DIR}/tools/deployment/certs.pem" DESTINATION "${CMAKE_BINARY_DIR}/package/pkg")
```
Comment (Smjert) [inline cmake/packaging.cmake]: Same

File: cmake/packaging.cmake (hunk 1 above)
Comment (theopolis) [reply cmake/packaging.cmake]: That makes sense. I was trying to follow the conventions above where `${CMAKE_SOURCE_DIR}/tools/deployment/osqueryd.service` and similar files are moved into a directory within the binary path.

Comment (Smjert) [reply cmake/packaging.cmake]: Yep sorry, that file needs a cleanup.

**Pull request merged**